src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -617,8 +601,7 @@ ParsedQuery Visitor::visit(Parser::SelectQueryContext* ctx) {
   if (!ctx->datasetClause().empty()) {
     // TODO: see if it is possible to extend reportError s.t. it can also take
     //  vector<ParserRuleContext>.
-    reportError(ctx->datasetClause(0),
-                "QLever currently doesn't support FROM clauses");
+    reportNotSupported(ctx->datasetClause(0), "FROM clauses are");
```

Comment:
I think just making this `visitVector<void>` (see above) would be the easiest way.